sorcery.lic

```diff
@@ -10,22 +10,35 @@ class Sorcery
   include DRCT
 
   def initialize
+    arg_definitions = [
+      [
+        { name: 'target_mindstate', regex: /\d+/, optional: true, description: 'Repeat the sorcery casting until reaching this mindstate.' }
+      ]
+    ]
+    args = parse_args(arg_definitions)
     @settings = get_settings
 
-    return if cast_sorcery_spell?
+    return if cast_sorcery_spell?(args.target_mindstate)
 
-    focus_runestone
+    focus_runestone(args.target_mindstate)
   end
 
-  def cast_sorcery_spell?
+  def cast_sorcery_spell?(target_mindstate)
     sorcery_spell = @settings.crossing_training_sorcery
     return false unless sorcery_spell
 
     walk_to(@settings.crossing_training_sorcery_room)
```

Comment:
I know it's technically unrelated to your change, but I noticed that this `walk_to` isn't in the right place. If you backfire such that you need healing, you won't run back to the sorcery room - instead you'll stay in the healing room. Can you move this into the loop below?